### PR TITLE
Centralized attempt splitting logic

### PR DESCRIPTION
**Pull request by boajer**
depends on https://github.com/oat-sa/extension-tao-outcome/pull/192

### CODE REVIEW

**Comment by gabrielfs7** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: scripts/update/Updater.php [hunk 1]
```diff
@@ -182,7 +182,6 @@ public function update($initialVersion)
             $this->setVersion('7.5.0');
         }
 
-        $this->skip('7.5.0', '8.7.4');
-
+        $this->skip('7.5.0', '9.0.0');
```
[the remark this answers is not in the document]
Comment (boajer) [reply scripts/update/Updater.php:185]: Only one, as previously  was  `8.7.4`

[the remark this answers is not in the document]
Comment (gabrielfs7) [reply scripts/update/Updater.php:185]: Yes, true. Only one, but this is not a major cause it is backward compatible and it is a bug fix. Should be `8.7.5`

[the remark this answers is not in the document]
Comment (boajer) [reply scripts/update/Updater.php:185]: I've bumped major as we've bumped major version of the dependency here https://github.com/oat-sa/extension-tao-outcomeui/pull/335/files#diff-096f34b8841df0fda7fda714e51755f8R46

[the remark this answers is not in the document]
Comment (gabrielfs7) [reply scripts/update/Updater.php:185]: ok, you are right! :) Thanks for the heads up!

**Pull request merged**